lib/Test2/Event/ParseError.pm

```diff
@@ -0,0 +1,20 @@
+package Test2::Event::ParseError;
+use strict;
+use warnings;
+
+our $VERSION = '0.000012';
+
+BEGIN { require Test2::Event; our @ISA = qw(Test2::Event) }
```

Comment:
I have no strong opinion here. I was just copying what was in the `test-more` repo's event code.